Makefile

```diff
@@ -0,0 +1,60 @@
+.DEFAULT_GOAL := help
```

Comment:
Update new command calls as in https://github.com/20tab/django-continuous-delivery/blob/main/Makefile